datastore/datastore.py

```diff
@@ -349,3 +354,44 @@ def exists(self):
             return elastic_search.create.exists(connection=self._client_or_connection, index_name=self._store_name)
 
         return False
+
+    def external_api_update_entity(self, dictionary_name, dictionary_data, language_script, **kwargs):
```

Comment:
Rename `dictionary_name` to `entity_name` at all places

Reply:
done